elyra/pipeline/component.py

```diff
@@ -255,7 +255,7 @@ def read_component_definition(self, component_id: str, location: str) -> str:
         component_location = os.path.join(os.path.dirname(__file__), location)
         if not os.path.exists(component_location):
             self.log.error(f'Invalid location for component: {component_id} -> {component_location}')
```

Comment:
If we're going to treat this in a less severe manner, I would recommend logging a warning message rather than error.
```suggestion
            self.log.warning(f'Invalid location for component: {component_id} -> {component_location}')
```